som-interpreter-bc/src/main.rs

```diff
@@ -33,15 +34,15 @@ struct Options {
     args: Vec<String>,
 
     /// Set search path for application classes.
-    #[structopt(long, short)]
+    #[clap(long, short)]
```

Comment:
Same suggestion as the one above.

```suggestion
    #[clap(long, short, multiple_values(true))]
```